src/main/java/org/opensearch/ad/caching/CacheBuffer.java

```diff
@@ -40,13 +42,13 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.opensearch.ad.ExpiringState;
-import org.opensearch.ad.MaintenanceState;
 import org.opensearch.ad.MemoryTracker;
 import org.opensearch.ad.MemoryTracker.Origin;
-import org.opensearch.ad.ml.CheckpointDao;
 import org.opensearch.ad.ml.EntityModel;
 import org.opensearch.ad.ml.ModelState;
 import org.opensearch.ad.model.InitProgressProfile;
+import org.opensearch.ad.ratelimit.CheckpointWriteQueue;
+import org.opensearch.ad.ratelimit.SegmentPriority;
```

Comment:
Are these classes defined somewhere? I can't find them in the current repo nor in this diff.

Reply:
Yes, they are defined somewhere and was included in separate PRs.  Please use this branch to check: https://github.com/kaituo/anomaly-detection-1/tree/multiCat